.nvmrc

```diff
@@ -0,0 +1,2 @@
+$ node -v
+v10.16.0
```

Comment:
this file just should have one line, with the version of node, it is not needed the node -v command

Reply:
I already corrected that.